lib/spack/spack/solver/asp.py

```diff
@@ -2856,13 +2878,22 @@ def reset(self):
         """Resets the current state."""
         self.current_package = None
 
-    def depends_on(self, dependency_str: str, *, when: str, type: str, description: str) -> None:
+    def depends_on(
+        self,
+        dependency_str: str,
+        *,
+        when: str,
+        type: str,
+        description: str,
+        languages: Optional[List[str]] = None,
+    ) -> None:
         """Injects conditional dependencies on packages.
 
         Args:
             dependency_str: the dependency spec to inject
             when: anonymous condition to be met on a package to have the dependency
             type: dependency type
+            languages: languages needed by the package for the dependency to be needed
```

Comment:
needs rewording -- I dunno what this means

Reply:
Clearly a rebasing issue or something. See https://github.com/spack/spack/pull/42062/commits/b0e97fb761ebf8e7f0f5b725538bf31b074803d6